### PR TITLE
build: switch to npm

### PR DESCRIPTION
**Pull request by denkristoffer**
🔔 This PR will not go green because our Vercel deploy command has hardcoded use of Yarn. I don't want to change it unless we decide to merge.

## Motivation

- Yarn v1 is unmaintained
- Yarn v1 has resolution bugs present since version `1.19` that you have to downgrade (or use `npx yarn@1.18`) to get around
- Yarn v1 doesn't have automatic dependency deduplication

I think these issues are grave enough that we either upgrade to Yarn v3 (which allows you not to use plug and play) or switch to npm which has had support for workspace for a long time. The primary reason I prefer switching to npm is that it ships with node, so it's easier to get started. I'm open to having these thoughts challenged, let me know if you prefer Yarn v3 🙂

### CODE REVIEW

**Review by massao — APPROVED** [review]
Looks good to me, unless we decide to update yarn, but maybe that could take more effort 🤔

**Review by mshaaban0 — APPROVED** [review]
Works like a champ 👍🏼

File: scripts/changesets/generate-releases.js [hunk 1]
```diff
@@ -61,10 +61,12 @@ async function main() {
   });
 
   // Run changesets publish and get stdout
-  const csOutput = childProcess.execSync('yarn changeset publish').toString();
+  const csOutput = childProcess
```
Comment (mshaaban0) [inline scripts/changesets/generate-releases.js:64]: eslint wants this in one line

File: .circleci/config.yml [hunk 2]
```diff
@@ -116,22 +111,21 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/circleci-f36-build
-      - node/install-packages:
-          pkg-manager: yarn
+      - node/install-packages
```
[the remark this answers is not in the document]
Comment (denkristoffer) [reply .circleci/config.yml:114]: It defaults to npm.

File: .circleci/config.yml [hunk 3]
```diff
@@ -14,11 +14,10 @@ jobs:
...
       - run:
           name: Build packages
-          command: yarn build
+          command: npm run-script build
```
[the remark this answers is not in the document]
Comment (denkristoffer) [reply .circleci/config.yml:20]: `run` is an alias for `run-script`. Why it's like that I don't know 🤷

[the remark this answers is not in the document]
Comment (mshaaban0) [reply .circleci/config.yml:20]: I see, I assume everyone is used to `run` but it's a small thing

**Pull request merged**